ambari-common/src/main/python/ambari_commons/os_linux.py

```diff
@@ -84,3 +84,7 @@ def os_set_open_files_limit(maxOpenFiles):
 
 def os_getpass(prompt):
   return getpass.unix_getpass(prompt)
+
+def os_is_service_exist(serviceName):
+  status = os.system("service %s status >/dev/null 2>&1" % serviceName)
```

Comment:
`service` command is kinda outdated and belongs to the `systemv`,  the thing which should be used - systemd, if system is controlled by it: `systemctl list-units --full -all | grep -Fq "$SERVICENAME.service"`

The problem is that service is not even preinstalled now in a fresh linux distros and may fail.   So we need to check, if system is systemd enabled, use `systectl`, and if system is old - use service

Reply:
ok, I'll fix it.